jms/src/main/scala/akka/stream/alpakka/jms/JmsConnector.scala

```diff
@@ -97,3 +122,41 @@ private[jms] case class JmsSession(connection: jms.Connection, session: jms.Sess
       }
     }
 }
+
+private[jms] class JmsAckSession(override val connection: jms.Connection,
+                                 override val session: jms.Session,
+                                 override val destination: jms.Destination,
+                                 val maxPendingAcks: Int)
+    extends JmsSession(connection, session, destination) {
+
+  private[jms] var pendingAck = 0
+  private[jms] val ackQueue = new ArrayBlockingQueue[() => Unit](maxPendingAcks + 1)
+
+  // def ack(message: jms.Message): Unit = ackQueue.put(message.acknowledge _)
```

Comment:
Why was this replaced with the same thing and commented out?

Reply:
Ahhh, thanks for the catch. This is actually the same as below. The below allowed me to add logging while debugging this work. Will reset it back.